neural_network/activation_functions/binary_step.py

```diff
@@ -0,0 +1,37 @@
+"""
+This script demonstrates the implementation of the Binary Step function.
+
+It's an activation function in which if the input to the
+activation function is greater than a threshold, the neuron is
+activated, else it is deactivated
```

Comment:
```suggestion
It's an activation function in which the neuron is activated if the input is
non-negative, otherwise it is deactivated
```
The binary step function specifies a threshold of 0